src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomColor() {
+
+        int ballNumber = new Random().nextInt(1, 100);

```

Comment:
```suggestion
        int ballNumber = random.nextInt(1, 101);
```
99 would be the biggest generated